diff --git a/build.gradle b/build.gradle
--- a/build.gradle
+++ b/build.gradle
@@ -730,17 +730,16 @@
 // We need to parse the second line
 def inStream = new java.io.BufferedReader(new java.io.InputStreamReader(new java.lang.ProcessBuilder(JAVA, "-version").start().getErrorStream()));
 try {
-    if (inStream.readLine() != null) {
-        String v = inStream.readLine();
-        if (v != null) {
-            int ib = v.indexOf(" (build ");
+    String v = inStream.readLine();
+    if (v != null) {
+            int ib = v.indexOf(" version ");
+
             if (ib != -1) {
                 String ver = v.substring(ib + 8, v.size() - 1);
 
                 defineProperty("jdkRuntimeVersion", ver)
                 defineProperty("jdkVersion", jdkRuntimeVersion.split("-")[0])
                 defineProperty("jdkBuildNumber", jdkRuntimeVersion.substring(jdkRuntimeVersion.lastIndexOf("-b") + 2))
-            }
         }
     }
 } finally {
